src/components/TextLink.js

```diff
@@ -33,7 +33,10 @@ const defaultProps = {
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
     return (
-        <Pressable
+        <Text
+            style={[styles.link, ...additionalStyles]}
+            accessibilityRole="link"
+            href={props.href}
```

Comment:
`href` doesn't seem to be a prop of our [Text](https://github.com/Expensify/App/blob/main/src/components/Text.js) component, I think it should be removed.